old_tests/invariances.py

```diff
@@ -1,89 +0,0 @@
-from jax.test_util import check_grads
-from scipy.stats import special_ortho_group
-import numpy as np
-import jax
-
-def assert_potential_invariance(energy_fn, x0, params, box=None):
-    # note: all potentials must support non-periodic implementations,
-    # defined by box being None
```

Comment:
Is it redundant to test translational and rotational invariance given the other ways the potentials are currently tested?

Reply:
Most of the old_tests were for an older implementation before we had PBCs. Unfortunately PBCs remove the rotational invariance of potential energy function for the nonbonded potential. (Bonded terms do not consider PBCs, ever, however, the exclusions *do* consider the PBCs). So this test needs to be re-written in a slightly more clever way.

The old_tests weren't really being run, and be done purely on the jax layer. So I'll probably add these back in later on.